packages/loader/container-loader/src/container.ts

```diff
@@ -354,6 +354,8 @@ export class Container
 
 		const container = new Container(createProps, loadProps);
 
+		const disableRecordHeapSize = container.mc.config.getBoolean("Fluid.Loader.recordHeapSize");
```

Comment:
the naming of this const is a bit confusing. It should either be `recordHeapSize`, or leave it `disableRecordHeapSize ` but then rename the feature flag to `Fluid.Loader.DisableRecordHeapSize` and below you can do:

```
	{ start: true, end: true, cancel: "generic" },
			disableRecordHeapSize !== true /* recordHeapSize */,
		);
```

because as of now, if you set the feature flag to true, `disableRecordHeapSize` will be true, which in turn will actually enable recording the heap size.

Reply:
Awesome! Great! Thank you!